web3-utils/script/geth.sh

```diff
@@ -0,0 +1,15 @@
+docker-compose build
+docker pull ethereum/client-go:stable
+docker-compose up -d deployer geth
+cargo test --no-run -- --ignored
+cd bus-mapping
+
+GethBlock=`curl localhost:8545 -X POST -H "Content-Type: application/json" -d '{"jsonrpc":"2.0","method":"eth_blockNumber","params":[],"id":1}'`
+
+while [ $GethBlock != '{"jsonrpc":"2.0","id":1,"result":"0x2"}' ]
+do
+    sleep 3
+    GethBlock=`curl localhost:8545 -X POST -H "Content-Type: application/json" -d '{"jsonrpc":"2.0","method":"eth_blockNumber","params":[],"id":1}'`
+done
+
+cargo test -- --ignored
```

Comment:
Why `-- --ignored`?

Here what we should do IMO is to run the integration tests only.
Which basically means, passing a feature flag that activates them and deactivates the others.

Reply:
I added `geth_rpc_test` feature!